web/client/src/App.js

```diff
@@ -4,6 +4,7 @@ import {
   TokenForm,
   ValidateToken,
   TokenDisplay,
+  ReqModule,
```

Comment:
Use full name for components: `RequestModule`

Reply:
Yep changed, but GitHub is down so cannot see it